modules/migrating-between-capi-mapi.adoc

```diff
@@ -0,0 +1,153 @@
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have deployed an {product-title} cluster on a supported infrastructure type.
+
+ifdef::machine-to-cluster[]
+* You have enabled the use of the Cluster API.
+endif::[]
+
+* You have enabled the `MachineAPIMigration` feature gate in the `TechPreviewNoUpgrade` feature set.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Identify the {from-api-name} resource that you want to migrate to a {to-api-name} resource by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc get <resource_kind> -n {origin-namespace}
+----
++
+--
+where `<resource_kind>` is one of the following values:
+
+`machine.{from-api-group}`:: The fully qualified name of the resource kind for a compute or control plane machine.
++
+[IMPORTANT]
+====
+The output of this command includes the control plane machines.
+Migrating a control plane machine in not supported.
+====
+
+`machineset.{from-api-group}`:: The fully qualified name of the resource kind for a compute machine set.
+--
+
+. Edit the resource specification by running the following command:
++
+[source,terminal]
+----
+$ oc edit <resource_name>
+----
++
+where `<resource_name>` is the fully qualified name of the {from-api-name} resource that you want to migrate to a {to-api-name} resource.
+
+. In the resource specification, update the value of the `spec.authoritativeAPI` field:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: <api_group>/<api_version> <1>
+kind: <resource_kind> <2>
+metadata:
+  name: <resource_name> <3>
+  [...]
+spec:
+  authoritativeAPI: {to-api-value} <4>
+  [...]
+status:
+  authoritativeAPI: {from-api-value} <5>
+  [...]
+----
+<1> The API group and version vary depending on the resource kind.
+<2> The resource kind varies depending on the resource kind.
+<3> The name of the resource that you want to migrate.
+<4> Specify the authoritative API that you want this resource to use.
+For example, to start migrating a {from-api-name} resource to the {to-api-name}, specify `{to-api-value}`.
+<5> The value for the current authoritative API.
+This value indicates which API currently manages this resource
+Do not change the value in this part of the specification.
+
+.Verification 
+
+* Check the status of the conversion by running the following command:
++
+[source,terminal]
+----
+$ oc -n {origin-namespace} get <resource_kind>/<resource_name> -o json | jq .status.authoritativeAPI
+---- 
++
+where `<resource_kind>/<resource_name>` is the fully qualified name of the {from-api-name} resource that you want to migrate to a {to-api-name} resource.
++
+--
+* While the conversion progresses, this command returns a value of `Migrating`.
+If this value persists for a long time, check your cluster logs for more information and to identify potential issues.
```

Comment:
Should we tell the users where to look for those logs? It's the `cluster-capi-operator` Deployment in the `openshift-cluster-api` namespace.